java/runtime/src/main/java/org/ray/runtime/AbstractRayRuntime.java

```diff
@@ -253,6 +254,11 @@ private TaskSpec createTaskSpec(RayFunc func, RayActorImpl actor, Object[] args,
       actorCreationId = returnIds[0];
     }
 
+    UniqueId actorCreationDummyObjectId = actor.getCreationDummyObjectId();
+    if (isActorCreationTask) {
+      actorCreationDummyObjectId = actorCreationId;
```

Comment:
This is confusing, but I think for actor creation tasks, the creation dummy object is actually set to nil.

Reply:
It should only be set for tasks that are going to execute on an actual actor, whereas the actor creation task actually executes on a normal worker that hasn't been assigned an actor ID yet.